phonebook-ui/src/app/app-routing.module.ts

```diff
@@ -1,11 +1,26 @@
-import { NgModule } from '@angular/core';
-import { Routes, RouterModule } from '@angular/router';
+import {NgModule} from '@angular/core';
+import {RouterModule, Routes} from '@angular/router';
+import {AuthLayoutComponent} from "./common/layout/auth-layout/auth-layout.component";
+import {RegisterPageComponent} from "./auth/register-page/register-page.component";
+import {ActivationPageComponent} from "./auth/activation-page/activation-page.component";
+import {PendingPageComponent} from "./auth/pending-page/pending-page.component";
 
 
-const routes: Routes = [];
+const routes: Routes = [
+  {
+    path: '', component: AuthLayoutComponent, children: [
+      // { path: '', redirectTo: '/login', pathMatch: 'full' },
+      {path: '', redirectTo: 'user/register', pathMatch: 'full'},
+      {path: 'user/register', component: RegisterPageComponent},
+      {path: 'user/activation', component: ActivationPageComponent},
+      {path: 'user/pending/:token', component: PendingPageComponent},
```

Comment:
make the last two routes vice versa. Meaning, user/activation/token and user/pending